core/arch/arm/plat-imx/platform_config.h

```diff
@@ -263,22 +262,16 @@
 
 /* emulated SRAM, at start of secure DDR */
 
-#define CFG_CORE_TZSRAM_EMUL_START	0x4E000000
-
-#define TZSRAM_BASE			CFG_CORE_TZSRAM_EMUL_START
-#define TZSRAM_SIZE			CFG_CORE_TZSRAM_EMUL_SIZE
+#define TZSRAM_BASE			0x4E000000
+#define TZSRAM_SIZE			(1 * 1024 * 1024)
```

Comment:
This discards `CFG_CORE_TZSRAM_EMUL_SIZE` as a external config directive for the relate imx platforms. It is a generic config, ... at least used by other platforms and referenced in the **arm.mk** script.

Reply:
There is no `#ifndef` check around `CFG_CORE_TZSRAM_EMUL_SIZE` so it cannot be specified externally, but this is an unnecessary change so I'll revert it.

Reply:
OK, I restored `CFG_CORE_TZSRAM_EMUL_SIZE`.